### PR TITLE
RHDEVDOCS-2882 Tracker for Bug 1942908 - mention syslog prerequisite

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.5+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-2882
- Direct link to doc preview: https://deploy-preview-34584--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external?utm_source=github&utm_campaign=bot_dp and ctrl+f for "You must have a logging server that is" and also "such as a syslog server"
- SME review: @btaani  
- QE review: @kabirbhartiRH 
- Peer review: @missmesss
- All reviews complete. Please merge now.

### CODE REVIEW

File: modules/cluster-logging-collector-log-forward-project.adoc [hunk 1]
```diff
@@ -11,7 +11,11 @@ To configure forwarding application logs from a project, create a `ClusterLogFor
 
 .Prerequisites
 
+<<<<<<< HEAD
```
Comment (missmesss) [inline modules/cluster-logging-collector-log-forward-project.adoc:14]: Looks like git diff sneaked in.

**Comment by missmesss** [comment]
@rolfedh LGTM except for that git diff piece only. Right now it messes with the titles in the 2nd part of the doc, the preview shows them as (for example):
= Forwarding logs using the legacy Fluentd method
But it'll be fixed automatically.

File: modules/cluster-logging-collector-log-forward-project.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-collector-log-forward-project.adoc]: Nice catch! Thanks!

**Review by otikhomi — APPROVED** [review]

**Pull request merged**